### ISSUE TITLE
VarunVats9 - I have got one doubt.

### ISSUE THREAD

**Issue by VarunVats9** [githubteacher/developers-jun-2016-americas#139]
How was I able to create a branch, is it because I am a collaborator ?

**Comment by hectorsector** [comment]
@VarunVats9 yes! Creating a branch is an option that's only available for collaborators. When you're not a collaborator, you'd have to use the [Fork & Pull](https://help.github.com/articles/fork-a-repo/) model.
